cmake/curl-options-exclusions.txt

```diff
@@ -0,0 +1,55 @@
+- RANDOM_FILE:FILEPATH=/dev/urandom
```

Comment:
Please remove minuses at the beginning of lines, options types and values. We need only options names.

Reply:
Removed minuses and spaces. Only options names are in file.